README.md

```diff
@@ -75,5 +81,5 @@ eksek 'echo Hello'
 ```
 
 ```ruby
-Esekuter.new 'echo Hello'
+Eksekuter.new('echo Hello').run { 'This goes to stdin.' }
```

Comment:
If you update the Eksekuter part, also the `eksek 'echo Hello'` example right above it should be adapted accordingly (as in: let's add the stdin stuff there as well).

Reply:
Où se trouve-t-il mon cerveau?

Reply:
Done